ipywidgets/widgets/domwidget.py

```diff
@@ -48,3 +49,28 @@ def _repr_keys(self):
         # We also need to include _dom_classes in repr for reproducibility
         if self._dom_classes:
             yield '_dom_classes'
+
+    def set_tabindex(self, i=0):
+        """Set tabindex for this DOM element.
+        NB: this method is here for completeness
+        but should be avoided for i>0.
+        (see https://developer.paciellogroup.com/blog/2014/08/using-the-tabindex-attribute/)
+
+        Parameters
+        ----------
+        i: integer
+            Order in the keyboard tabulation.
+        """
+        self.tabindex = i
+
+    def set_tabbable(self):
```

Comment:
When I see a method `set_x`, I'm assuming it is going to set the value of `x`. Maybe call is `make_tabbable`? Maybe we should instead add a `tabbable = Bool()` trait, thereby avoiding the `tabindex > 0` case all together?

Reply:
ok for `make_` which is more consistent.

About avoiding the `tabindex > 0` : I'll have to ask an accessibility specialist.

Reply:
see https://github.com/jupyter-widgets/ipywidgets/pull/2640